### PR TITLE
Add Gluon (fka UnUniFi)

### PR DESCRIPTION
**Pull request by Senna46**
UnUniFi has been rebranded to Gluon.
UnUniFi will remain as is, and will be updated this in the future.

### CODE REVIEW

**Comment by JeremyParish69** [comment]
Looking at the version history, we can see this is an update of ununify, so I think we should not create anew gluon directory, but just update the branding for ununify.

**Comment by Senna46** [comment]
@JeremyParish69  Currently, there is a mainnet for UnUniFi and a testnet for Gluon. 
We would therefore like to register both; UnUniFi will be removed later.

**Comment by JeremyParish69** [comment]
> @JeremyParish69 Currently, there is a mainnet for UnUniFi and a testnet for Gluon. We would therefore like to register both; UnUniFi will be removed later.

Is gluon a testnet for ununify?

File: gluon/chain.json [hunk 1]
```diff
@@ -0,0 +1,455 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "gluon",
+  "status": "live",
+  "network_type": "mainnet",
```
Comment (JeremyParish69) [inline gluon/chain.json:5]: You have network type here is mainnet. Is this a mainnet or a testnet?

Comment (JeremyParish69) [reply gluon/chain.json:5]: or is there two, simultaneously running mainnets?

**Comment by JeremyParish69** [comment]
Is there still interest in proceeding with this @Senna46 ?

**Pull request closed without merging**